### DISCUSSION TITLE
Editing Original Photo

### DISCUSSION THREAD

**Discussion by Samlane86** [ente-io/ente#2346] [Enhancements]
Would it be possible to have the option to edit the original photo instead of saving a copy? I had a few pictures that I just wanted to flip to orient properly, but saving a copy removes a lot of the EXIF data.

**Comment by gzurbach** [comment]
This! Not only it creates a copy, but the Exif data is lost 🤷‍♂️ So the new photo looks like it was taken today. 

So if one day I decide to migrate to another photo service and export all my data, the information will be lost (except for the JSON file which is not practical).

**Comment by Weegee** [comment]
Adding to this: the menu option "save copy" implies to a user that an actual copy of the file is being saved. By removing EXIF data without asking, this is obviously not a direct copy, but an altered file. For someone not immediately checking their new files and noticing the missing EXIF data, this is unfortunately misleading.

**Comment by bobhett** [comment]
Agreed - this could be a deal-breaker for me with Ente.  It seems to work as expected on the Android app, but not the Website or Desktop App (Linux).  Why would I ever want the metadata to be removed due to a simple edit like rotate or crop?

There should be both a "Save" and "Save a Copy" options and they should both preserve the metadata

**Comment by chameleon-tbn** [comment]
Yeah, all mentioned in this thread is atm my biggest issue and show-stopper for ente.
